logging.conf

```diff
@@ -0,0 +1,29 @@
+[loggers]
+keys=root,__main__
+
+[handlers]
+keys=consoleHandler
+
+[formatters]
+keys=basicFormatter
+
+[logger_root]
+level=INFO
+handlers=consoleHandler
+qualname=root
+propagate=0
+
+[logger___main__]
+level=DEBUG
+handlers=consoleHandler
+qualname=__main__
+propagate=0
+
+[handler_consoleHandler]
+class=StreamHandler
+level=DEBUG
+formatter=basicFormatter
+args=(sys.stdout,)
+
+[formatter_basicFormatter]
+format=%(asctime)s loglevel=%(levelname)-6s logger=%(name)s %(funcName)s() L%(lineno)-4d %(message)s
```

Comment:
Can we remove the line number - the log message should tell us everything we need to know.